jib-gradle-plugin/README.md

```diff
@@ -183,7 +183,7 @@ Field | Type | Default | Description
 
 Property | Type | Default | Description
 --- | --- | --- | ---
-`image` | `String` | `gcr.io/distroless/java` | The image reference for the base image.
+`image` | `String` | `gcr.io/distroless/java` | The image reference for the base image. The source type can be specified using a [URL scheme](#setting-the-base-image).
```

Comment:
Perhaps there's no harm to avoid the term "URL scheme". "Special type prefixes"?

Reply:
Oh right, forgot to remove this